src/pages/index.tsx

```diff
@@ -46,23 +48,27 @@ export default function Home({
 
             <div className="flex flex-1 flex-col justify-between col-span-12 lg:col-span-4 space-y-4">
               <div className="space-y-3 bg-[#F1F1EF] p-6 rounded-3xl">
-                <h2 className="text-4xl font-semibold">40,000+</h2>
-                <span className="block text-lg">
-                  Death toll from the lates official figures
-                </span>
+                <Link href="https://en.afad.gov.tr/press-bulletin-32-about-the-earthquake-in-kahramanmaras" target="_blank">
+                  <h2 className="text-4xl font-semibold">40,000+</h2>
+                  <span className="block text-lg">
+                    Death toll from the t official figures.
```

Comment:
@emotemot in the text "the t official", is it "t" or is it a typo?

Reply:
@borabaloglu typo.